REV20231005F/alternate-components.md

```diff
@@ -0,0 +1,19 @@
+# Alternate Components
+
+**A table of acceptable replacement components for the AMDC REV20231005F**
+
+##
+
+| Original Part Number  | Alternate Part Number | Description      |  Designator  |
+|-----------------------|-------------|----------------------------|--------------|
+| [1528-1697-ND](https://www.digikey.com/en/products/detail/adafruit-industries-llc/3083/6058478?s=N4IgTCBcDaIIwFYwA4C0cBsBOA7KgcgCIgC6AvkA)          | [684-ATS014014007-SF-3F-ND](https://www.digikey.com/en/products/detail/advanced-thermal-solutions-inc/ATS014014007-SF-3F/18711115?s=N4IgTCBcDaIGwA4AsBaAggFQMoAYCMS%2BhOA7ClgGIoDMVAcgCIgC6AvkA)   | Heat Sink BGA Aluminum Top Mount  | HEATSINK1 |
+| | | |
+| [493-2203-6-ND](https://www.digikey.com/en/products/detail/nichicon/UWT1V101MCL1GS/589944?s=N4IgTCBcDaICwE4DMBaMYAMqBsKByAIiALoC%2BQA)         | [493-6338-1-ND](https://www.digikey.com/en/products/detail/nichicon/UWZ1V101MCL1GS/2550909?s=N4IgTCBcDaICwE4DMBaAbEpAOFBGFAcgCIgC6AvkA) | CAP ALUM 100UF 20% 35V SMD            | C80 |
+| | | |
+| [102-PQQ6W-Q24-S24-S-ND](https://www.digikey.com/en/products/detail/cui-inc/PQQ6W-Q24-S24-S/13402043?s=N4IgTCBcDaIIwAYwFoAKBFdA2A6s9YALMgMpGnIByAIiALoC%2BQA)| [2725-URB2424S-6WR3-ND](https://www.digikey.com/en/products/detail/mornsun-america-llc/URB2424S-6WR3/13168170?s=N4IgTCBcDa4OxgKwFoCqAlAQmALLgysgGwDq6AzMgHIAiIAugL5A) | 	DC DC CONVERTER 24V 6W   | U7 |
+|                       | [4457-PDL06-24S24W-ND](https://www.digikey.com/en/products/detail/p-duke-technology/PDL06-24S24W/13531900?s=N4IgTCBcDaICxwKwHYC0AFAIgGQAwDZUw4BlYgdVQDlMQBdAXyA)                   | 	 DC DC CONVERTER 24V 6W | U7 |
+| | | |
+| [SI4946CDY-T1-GE3DKR-ND](https://www.digikey.com/en/products/detail/vishay-siliconix/SI4946CDY-T1-GE3/7915484?s=N4IgTCBcDaIMoEkAsBOJA2AwgEQJoFoAVARnwHEBRAZmwGkAlfAOWxAF0BfIA)| [SI4900DY-T1-GE3CT-ND](https://www.digikey.com/en/products/detail/vishay-siliconix/SI4900DY-T1-GE3/2622306?s=N4IgTCBcDaIMoEkAsBOADGgIgTQLQBUBGXAcQFEBmAYX1wDlMQBdAXyA)  |  	MOSFET 2N-CH 60V 5.3A 8SOIC      | Q3 |
```

Comment:
This replacement is correct. Please also list this other part (yellow highlight below) that was approved in [this discussion post](https://github.com/Severson-Group/AMDC-Hardware/discussions/229#discussioncomment-7320717).

![image](https://github.com/user-attachments/assets/3186148e-ae31-4c30-95e1-9760e85ff53e)